Shared/Samples/Run valve isolation trace/RunValveIsolationTraceView.Enums.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extension RunValveIsolationTraceView {
+    /// The different states of a utility network trace.
+    enum TracingActivity: CaseIterable {
+        case loadingServiceGeodatabase, loadingNetwork, startingLocation, runningTrace, none
+        
+        /// A human-readable label for the tracing activity.
+        var label: String {
```

Comment:
Or you can conform to `CustomStringConvertible`.  Then instead of `label`, you would name this description

Reply:
That's not really what `CustomStringConvertible` is intended for. It is really intended for type-conversion (type->string), as the documentation for that protocol explains. That is most clearly demonstrated by `LosslessStringConvertible`, which refines `CustomStringConvertible` and affords converting the other direction (string->type). The samples app isn't localized, but we want to model good practices. It wouldn't be good practice to use a localized string as a string representation of an instance, particularly because that is what is used when the value is printed to the console. The string representation shouldn't change based on the language/region in which the app is being run. Therefore a custom computed property like this is actually the preferred approach. And to clarify that the type is intended for display to the user, the type of the property could be `LocalizedStringKey`, which is the type used when calling `Text.init(_:)` with a string literal.

Reply:
Thanks @philium, those are great points.  @des12437, sorry for misleading you with this particular PR feedback.

Reply:
Thank you @philium, I removed the `CustomStringConvertible` conformance.